pom.client.xml

```diff
@@ -494,33 +494,40 @@
               site for more Java documentation, including quick starts, tutorials, and code samples.
             </footer>
             <linksource>false</linksource>
-            <excludePackageNames>com.microsoft.azure.template</excludePackageNames>
+            <excludePackageNames>com.microsoft.azure.template:*.impl.*:*.implementation.*</excludePackageNames>
             <groups>
               <group>
                 <title>Azure Batch</title>
                 <packages>com.microsoft.azure.batch*</packages>
               </group>
-              <group>
-                <title>Azure Key Vault</title>
-                <packages>com.microsoft.azure.keyvault*</packages>
-              </group>
               <group>
                 <title>Azure Event Hubs</title>
                 <packages>com.microsoft.azure.eventhubs*:com.microsoft.azure.eventprocessorhost</packages>
               </group>
+              <group>
+                <title>Azure Key Vault</title>
+                <packages>com.microsoft.azure.keyvault*</packages>
+              </group>
               <group>
                 <title>Azure Common</title>
-                <packages>com.microsoft.azure.common.azure-common*</packages>
+                <packages>com.azure.common:com.azure.common.annotations:com.azure.common.credentials:com.azure.common.exception:com.azure.common.http*</packages>
```

Comment:
Why do we separate include different packages?

Reply:
Because otherwise all the auth and mgmt packages will be included also.